README.md

```diff
@@ -1,9 +1,6 @@
 # Mentorship System (Backend)
 
-| Branch | [Travis](https://travis-ci.org/) | [Codacy](https://www.codacy.com/) | Server |
-| :---: | :---: | :---: | :---: |
-| [master](https://github.com/anitab-org/mentorship-backend/tree/master) | [![Build Status](https://travis-ci.org/systers/mentorship-backend.svg?branch=master)](https://travis-ci.org/anitab-org/mentorship-backend) | [![Codacy Badge](https://api.codacy.com/project/badge/Grade/5181ca06835045d1aea78fdd30fc37d9?branch=master)](https://www.codacy.com/app/systers/mentorship-backend) | [Production](http://systers-mentorship.eu-central-1.elasticbeanstalk.com/) |
-| [develop](https://github.com/anitab-org/mentorship-backend/tree/develop) | [![Build Status](https://travis-ci.org/anitab-org/mentorship-backend.svg?branch=develop)](https://travis-ci.org/anitab-org/mentorship-backend) | [![Codacy Badge](https://api.codacy.com/project/badge/Grade/5181ca06835045d1aea78fdd30fc37d9?branch=develop)](https://www.codacy.com/app/systers/mentorship-backend) | [Development](https://mentorship-backend-temp.herokuapp.com/) |
+![Build Status](https://github.com/anitab-org/mentorship-backend/workflows/Run%20tests/badge.svg)
```

Comment:
this is the badge for run tests only right? Build is another badge? Also could you get a badge for codecov?

Reply:
Are we not using Codacy anymore 🤔 I actually am not sure cc @mayburgos

Reply:
Hey @isabelcosta the travis script just runs the tests and then publishes on aws as I directly converted the travis script into gh actions :sweat_smile:

Reply:
The codacy thing I am not sure about @isabelcosta :sweat_smile: I figured I didn't see the test running so we might not be using it but not sure